config/checkstyle/suppressions.xml

```diff
@@ -7,4 +7,8 @@
 <suppressions>
     <suppress checks="FileLength"
               files="DefaultBeanContext.java|BeanDefinitionWriter.java|DefaultHttpClient.java"/>
+    <suppress checks="DeclarationOrder"
+              files="UriTemplate.java"/>
+    <suppress checks="Header"
+              files="AnnotationClassReader.java|Attribute.java|Context.java|io[\\/]micronaut[\\/]core[\\/]util[\\/]clhm|AmazonInfo.java|DataCenterInfo.java|LeaseInfo.java|QueryStringDecoder.java|CronExpression.java|AntPathMatcher.java|io[\\/]micronaut[\\/]http[\\/]netty[\\/]stream[\\/]package-info.java" />
```

Comment:
All the items in this list are classes that appear to be copied in from other projects and have the license from the other project (Apache, Google, etc). Sometimes I could exclude by package b/c all the files were not ours other times the packages had a mixture so just went after class name.